opa/views/course_views.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="course_tree_view" model="ir.ui.view">
+        <field name="name">course.tree.view</field>
+        <field name="model">course</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="title"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="course_search_view" model="ir.ui.view">
+        <field name="name">course.search.view</field>
+        <field name="model">course</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="title"/>
+                <field name="description"/>
+                <filter name="my_courses" string="My Courses" domain="[('responsible_id', '=', uid)]"/>
+                <group string="Group By">
+                    <filter name="group_by_responsible" string="Responsible" context="{'group_by': 'responsible_id'}"/>
+                </group>
+            </search>
+        </field>
+    </record>
+
+    <record id="course_form_view" model="ir.ui.view">
+        <field name="name">course.form.view</field>
+        <field name="model">course</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <group>
+                        <field name="title"/>
+                    </group>
+                    <notebook>
+                        <page string="Description">
+                            <field name="description"/>
+                        </page>
+                        <page string="Sessions">
+                            <field name="session_ids"/>
```

Comment:
`nolabel` not required?

Reply:
`description` is not inside a group so labels aren't added automatically if that is what you meant.